README.md

```diff
@@ -2,56 +2,80 @@
 
 <br/>  
 <div align="center" style="display:flex;">  
-  <img width="600" src="https://user-images.githubusercontent.com/6308023/165422882-2ab14b87-7370-4e3a-9a01-94b545a26316.png">
+  <img width="400" alt="v_black_square_800px" src="https://user-images.githubusercontent.com/63234878/209922215-cccf7b88-de12-42ac-9714-8a5e3f194d7f.png">
+
   <p> 
     <br>
-    <img alt="Version"  src="https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000"  />    
+    <img alt="Version"  src="https://img.shields.io/badge/version-v1.0--beta--rc.1-blue.svg?cacheSeconds=2592000"  />    
     <a href="https://www.apache.org/licenses/LICENSE-2.0"  target="_blank"><img alt="License: Apache 2.0"  src="https://img.shields.io/badge/License-Apache 2.0-yellow.svg" /></a> 
   </p> 
-</div>    
+</div> 
 
-## 1. About VATZ ##
+# What is VATZ?
 
-> **VATZ** is a tool for building, analyzing, and managing blockchain Node infrastructure safely and efficiently. You can set up VATZ to manage existing or new blockchain nodes and integrate with popular services PagerDuty, Discord, and more as well as custom in-house solutions.
+**VATZ** is a tool for building, analyzing, and managing blockchain node infrastructure safely and efficiently. You can set up VATZ to manage existing or new blockchain nodes and integrate with popular services PagerDuty, Discord, and more as well as custom in-house solutions.
 
-The key features of **VATZ** are:
 
-- `Multi Protocol Support`: **VATZ** is NOT limited Protocol Type where it categorizes on chain protocol. Any Protocol can be managed through VATZ with plugins, even unsupported protocols can be integrated through simple plugin development.
-- `Infrastructure as Code`: **VATZ** is described using a high-level configuration syntax. You can divide your plugins into modular components that can then be combined in different ways to behave through automation.
-- `Data Analysis`: **VATZ** helps to build datasets for your managing protocols and transfer your data into popular services Prometheus, Kafaka, Google BigQuery and more. Because of this, VATZ aims to set your Node infrastructure as efficiently as possible, and operators get insight into dependencies in their infrastructure. (2023-Q3)
-- `Change automation`: Complex sets of node's operational tasks can be done through **VATZ** with minimal human interaction. (2023-Q2)
 
+## How does **VATZ Project** work?
 
-#### _To find more about_ **`VATZ Project`**
-- [VATZ Project Design](docs/design.md)
+VATZ project is primarily designed to check the node states in real-time and receive alert notifications of all blockchain protocols, including metrics that the protocol itself does not support. 
 
-#### _To get start with_ **VATZ**
-- [Installation guide](docs/installation.md)
+To this end, it consists of 3 components:
 
```

Comment:
@qkrybin 
Can you add here?
#### **VATZ** _CLI Command list_

@rootwarp 
Can you put the link of CLI command md.file here once you create a file.

Reply:
@xellos00 
Can I add the command list in the 'usage' part?

Reply:
> @xellos00 Can I add the command list in the 'usage' part?

Ah, Yeah. you are right. 
It should be in the 'usage' part.